src/main/java/ailurus/task/TaskList.java

```diff
@@ -0,0 +1,107 @@
+package ailurus.task;
+
+import ailurus.AilurusException;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> list;
+
+    /**
+     * Constructor for empty TaskList
+     */
+    public TaskList() {
+        this.list = new ArrayList<>();
+    }
+
+    /**
+     * Constructor for Tasklist with default tasks
+     *
+     * @param tasks default tasks
+     */
+    public TaskList(ArrayList<Task> tasks) {
+        this.list = new ArrayList<>(tasks);
+    }
+
+    /**
+     * Size of arraylist of tasks
+     *
+     * @return length of arraylist
+     */
+    public int length() {
+        return this.list.size();
+    }
+
+    /**
+     * Get the task message of a particular task
+     *
+     * @param i index of the task
+     * @return task message of the task
+     */
+    public String getIndexString(int i) {
+        return list.get(i).toString();
+    }
+
+    /**
+     * Get a particular task from the task list
+     *
+     * @param i index of the task
+     * @return the particular task
+     */
+    public Task getTask(int i) {
+        return list.get(i);
+    }
+
```

Comment:
I think it violets tell don't ask principle, may be you can find what should be done to tasks, like compare task times, compare two tasks content, they can be done using a method defined in tasklist class.